app/services/approve_appeal_service.rb

```diff
@@ -53,7 +53,7 @@ def undo_delete_statuses!
 
   def undo_mark_statuses_as_sensitive!
     representative_account = Account.representative
-    @strike.statuses.includes(:media_attachments).find_each do |status|
+    @strike.statuses.kept.includes(:media_attachments).reorder(nil).find_each do |status|
```

Comment:
Side issue here, but I wonder if we could remove the ORDER portion of the default scope on Status? The `kept` part should probably stay because that seems very likely to be forgotten if not a default, but the order ... maybe eventually?

Reply:
That's a wider change than what I am ok with in this PR but it *might* make sense in the future.